src/absil/il.fsi

```diff
@@ -751,50 +751,69 @@ type ILAttribElem =
 /// Named args: values and flags indicating if they are fields or properties.
 type ILAttributeNamedArg = string * ILType * bool * ILAttribElem
 
+
 /// Custom attributes.  See 'decodeILAttribData' for a helper to parse the byte[] 
-/// to ILAttribElem's as best as possible.  
-type ILAttribute =
-    { Method: ILMethodSpec  
-      Data: byte[] 
-      Elements: ILAttribElem list}
+/// to ILAttribElem's as best as possible.
+type IAttribute =
```

Comment:
Please keep  the names as `ILAttribute` etc.

Reply:
I guess the point here is to honor .NET guidelines which say interface names should start with `I`.
This would lead to `IILAttribute`

Reply:
Please just use `ILAttribute`, i.e. the same names as now. Thanks

Reply:
> I guess the point here is to honor .NET guidelines which say interface names should start with I.

Yes, the idea was to follow these guidelines.

@dsyme OK, I'll change the names this way. I do, however, really like idea of using more conventional names. Would you mind returning to the naming discussion at some later point and probably having this stuff renamed in a future PR?